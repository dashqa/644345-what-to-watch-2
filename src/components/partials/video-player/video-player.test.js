import React from 'react';
import renderer from 'react-test-renderer';
import VideoPlayer from './video-player';

const props = {
  poster: ``,
  link: ``,
  width: ``,
  height: ``,
  controls: false,
  muted: false,
};

it(`Video player correctly renders after relaunch`, () => {
  const tree = renderer.create(<VideoPlayer {...props}/>).toJSON();
  expect(tree).toMatchSnapshot();
});
