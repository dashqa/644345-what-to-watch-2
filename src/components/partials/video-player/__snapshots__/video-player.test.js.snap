// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Video player correctly renders after relaunch 1`] = `
<video
  className=""
  controls={false}
  height=""
  muted={false}
  poster=""
  width=""
>
  <source
    src=""
    type="video/mp4"
  />
</video>
`;
